.gitignore

```diff
@@ -1,3 +1,3 @@
-/nginx/servers/*
-!/nginx/servers/proximis.conf.dist
```

Comment:
i would prefer to keep `servers`, to get a common structure across all the docker projects that we have. And this is not going to be a BC break since it will be tag `4.0.0` when it will be merged in master. @ajardin what do you think ?